root/static/scripts/common/components/FilterForm.js

```diff
@@ -92,6 +101,19 @@ const FilterForm = ({form}: Props): React.Element<'div'> => {
               </tr>
             ) : null}
 
+            {showACDroppedMessage && showArtistCreditForm ? (
+              <tr>
+                <td />
+                <td>
+                  {exp.l(
+                    `<strong>Warning:</strong> The artist credit filter
+                     was dropped because the list of available artist credits
+                     changed.`,
+                  )}
+                </td>
+              </tr>
+            ) : null}
+
```

Comment:
While this might be slightly clearer, I'm not sure it's worth introducing state into this form just for it. Am I missing a way to do this without state / hydration?

Reply:
I don’t know if any of these are required at all or if they will be required in the future anyway. Maybe @mwiencek has a longer-term plans for this component?